src/estimators/exact_set.py

```diff
@@ -33,18 +35,18 @@ def f(random_seed):
     return f
 
   def __init__(self, random_seed=None):
-    """Construct an ExactSet sketch.
+    """Construct an ExactMultiSet sketch.
 
     Args:
       random_seed: This arg exists in order to conform to
         simulator.EstimatorConfig.sketch_factory.
     """
     SketchBase.__init__(self)
     _ = random_seed
-    self._ids = set()
+    self._ids = {}
```

Comment:
It doesn't matter, but this could be a collections.Counter object, which basically all it does is automatically do obj.get(key, 0) when you do obj[key] += 1

Reply:
I actually looked at that as a choice, but in the end I concluded it didn't really save anything.  But thanks for the suggestion!